django/chat/static/chat/js/scripts.js

```diff
@@ -519,3 +519,13 @@ document.addEventListener("DOMContentLoaded", function () {
     }
   });
 });
+
+
+function updatePageTitle(title = null) {
```

Comment:
Handles first page load, title update after streaming response, and manual renaming